data/welcome.json

```diff
@@ -1,4 +1,11 @@
 {
     "it": ["Benvenuto USER nel nostro gruppo ^-^","Ciao USER!","Salve USER!"],
-    "en": ["Welcome USER to our group ^-^","Hello USER!","Howdy USER!"]
+    "en": ["Welcome USER to our group ^-^","Hello USER!","Howdy USER!"],
+    "readme": "https://t.me/c/1095167198/67194",
+    "utils": [
```

Comment:
Al fine di mantenere il supporto alla doppia lingua (e ulteriori future) penso possa essere utile trasformare le liste contenute in `it` ed `en` in dizionari e incorporare all'interno di questi nuovi dizionari `utils` in entrambe le lingue